src/test/unit_tests/jest_tests/test_nc_nsfs_account_cli.test.js

```diff
@@ -372,6 +381,52 @@ describe('manage nsfs cli account flow', () => {
             expect(JSON.parse(res.stdout).error.message).toBe(ManageCLIError.InvalidArgument.message);
         });
 
+        it('cli list filter by user (account 4)', async () => {
```

Comment:
Could you please add test for bucket aslo.

Reply:
added thanks